diff --git a/EarTrumpet/App.xaml b/EarTrumpet/App.xaml
index ef70204..5ef2ca9 100644
--- a/EarTrumpet/App.xaml
+++ b/EarTrumpet/App.xaml
@@ -451,6 +451,7 @@
                 <Setter Property="Maximum" Value="100" />
                 <Setter Property="IsTabStop" Value="True" />
                 <Setter Property="Focusable" Value="True" />
+                <Setter Property="FocusVisualStyle" Value="{x:Null}" />
                 <Setter Property="SmallChange" Value="1" />
                 <Setter Property="LargeChange" Value="10" />
                 <Setter Property="SnapsToDevicePixels" Value="True" />
diff --git a/EarTrumpet/UI/Controls/BalanceSlider.cs b/EarTrumpet/UI/Controls/BalanceSlider.cs
index 63e01b8..d4a8374 100644
--- a/EarTrumpet/UI/Controls/BalanceSlider.cs
+++ b/EarTrumpet/UI/Controls/BalanceSlider.cs
@@ -126,6 +126,7 @@ namespace EarTrumpet.UI.Controls
             _isSnapped = false;
             CaptureMouse();
             UpdateValueFromPoint(e.GetPosition(this));
+            Focus();
             e.Handled = true;
         }
 
