diff --git a/EarTrumpet/UI/Controls/BalanceSlider.cs b/EarTrumpet/UI/Controls/BalanceSlider.cs
index d4a8374..63e01b8 100644
--- a/EarTrumpet/UI/Controls/BalanceSlider.cs
+++ b/EarTrumpet/UI/Controls/BalanceSlider.cs
@@ -126,7 +126,6 @@ namespace EarTrumpet.UI.Controls
             _isSnapped = false;
             CaptureMouse();
             UpdateValueFromPoint(e.GetPosition(this));
-            Focus();
             e.Handled = true;
         }
 
diff --git a/EarTrumpet/App.xaml b/EarTrumpet/App.xaml
index ef70204..1094cae 100644
--- a/EarTrumpet/App.xaml
+++ b/EarTrumpet/App.xaml
@@ -449,8 +449,8 @@
                 <Setter Property="OverridesDefaultStyle" Value="True" />
                 <Setter Property="Minimum" Value="-100" />
                 <Setter Property="Maximum" Value="100" />
-                <Setter Property="IsTabStop" Value="True" />
-                <Setter Property="Focusable" Value="True" />
+                <Setter Property="IsTabStop" Value="False" />
+                <Setter Property="Focusable" Value="False" />
                 <Setter Property="SmallChange" Value="1" />
                 <Setter Property="LargeChange" Value="10" />
                 <Setter Property="SnapsToDevicePixels" Value="True" />
diff --git a/EarTrumpet/UI/Views/DeviceView.xaml b/EarTrumpet/UI/Views/DeviceView.xaml
index ca040ff..7d010cc 100644
--- a/EarTrumpet/UI/Views/DeviceView.xaml
+++ b/EarTrumpet/UI/Views/DeviceView.xaml
@@ -166,7 +166,6 @@
                                 Minimum="-100"
                                 Maximum="100"
                                 IsSnapToTickEnabled="False"
-                                IsTabStop="True"
                                 SmallChange="1"
                                 LargeChange="10"
                                 Value="{Binding Device.Balance, Mode=TwoWay}"
