diff --git a/EarTrumpet/App.xaml b/EarTrumpet/App.xaml
index 1094cae..ef70204 100644
--- a/EarTrumpet/App.xaml
+++ b/EarTrumpet/App.xaml
@@ -449,8 +449,8 @@
                 <Setter Property="OverridesDefaultStyle" Value="True" />
                 <Setter Property="Minimum" Value="-100" />
                 <Setter Property="Maximum" Value="100" />
-                <Setter Property="IsTabStop" Value="False" />
-                <Setter Property="Focusable" Value="False" />
+                <Setter Property="IsTabStop" Value="True" />
+                <Setter Property="Focusable" Value="True" />
                 <Setter Property="SmallChange" Value="1" />
                 <Setter Property="LargeChange" Value="10" />
                 <Setter Property="SnapsToDevicePixels" Value="True" />
